src/main/scala/scalismo/io/StatisticalModelIO.scala

```diff
@@ -19,24 +19,58 @@ import java.io.File
 
 import scalismo.common.{ PointId, UnstructuredPointsDomain }
 import scalismo.geometry.{ Point, _3D }
-import scalismo.io.StatismoIO.StatismoModelType.StatismoModelType
 import scalismo.mesh.{ TriangleCell, TriangleList, TriangleMesh, TriangleMesh3D }
 import scalismo.mesh.TriangleMesh._
 import scalismo.statisticalmodel.StatisticalMeshModel
 
 import scala.util.Try
 import breeze.linalg.DenseVector
 import breeze.linalg.DenseMatrix
+
 import scala.util.Failure
 import scala.util.Success
 import java.util.Calendar
+
 import ncsa.hdf.`object`._
 import java.io.DataOutputStream
 import java.io.FileOutputStream
 import java.io.DataInputStream
 import java.io.FileInputStream
 
+import scalismo.io.StatismoIO.StatismoModelType.StatismoModelType
+
+object StatisticalModelIO {
```

Comment:
I agree that `StatismoIO` is not the most straight forward name to use when regarding the rest of the naming of the library. Is there a reason to not call this `StatisticalMeshModelIO` with the methods `read` and `write`? For me, this would a more straight forward naming scheme.